letsencrypt/cli.py

```diff
@@ -466,8 +466,9 @@ def auth(args, config, plugins):
 
     # This is a special case; cert and chain are simply saved
     if args.csr is not None:
+        csr = read_file(args.csr)
```

Comment:
Also: can we use `csr_path, csr_contents = read_file(...)` to make code more readable?